src/lib/dragondeploy/arns.ts

```diff
@@ -47,13 +49,15 @@ export async function registerArNSName({
   transactionId: string
 }) {
   name = name.toLowerCase()
-  const registry = getWarpPstContract(REGISTRY, window.arweaveWallet)
-  const owner = await window.arweaveWallet.getActiveAddress()
+
+  const signer = await getSigner()
+  const registry = getWarpPstContract(REGISTRY, signer)
+  const owner = useGlobalStore.getState().authState.address!
 
   // create ANT contract
   const ant = await warp.createContract.deployFromSourceTx(
     {
-      wallet: window.arweaveWallet,
```

Comment:
Its should be window.arweaveWallet for ArConnect when bundling is disabled otherwise it throw error as `* Only ArWallet | CustomSignature wallet type are allowed when bundling is disabled.` Also, same error is thrown for othent as well.

Reply:
fixed this